src/Sylius/Behat/Context/Transform/ProductContext.php

```diff
@@ -22,9 +22,13 @@ final class ProductContext implements Context
     /** @var ProductRepositoryInterface */
     private $productRepository;
 
-    public function __construct(ProductRepositoryInterface $productRepository)
+    /** @var string */
+    private $locale;
+
+    public function __construct(ProductRepositoryInterface $productRepository, string $locale = 'en_US')
```

Comment:
As there is no BC promise on our tests, maybe we can just skip default value here?